src/plugins/legacy/reformat/resliceToolBox.cpp

```diff
@@ -263,6 +263,7 @@ void resliceToolBox::updateView()
 {
     medAbstractView* view = this->getWorkspace()->tabbedViewContainers()->getFirstSelectedContainerView();
 
+    d->currentView = nullptr;
     if (view && d->currentView!= view)
```

Comment:
The second part of this test should be removed

Reply:
Indeed, it's done, i'm going to add this change to the other PR also